### DISCUSSION TITLE
storage_class standard= rrs=EC:1

### DISCUSSION THREAD

**Discussion by Misszhangruihua** [minio/minio#18913] [Q&A] [answered]
This is my cluster configuration information
![image](https://github.com/minio/minio/assets/86996849/9f05eb01-fe2d-412a-bdc7-3190425fa1de)
![image](https://github.com/minio/minio/assets/86996849/f2c45ba6-2410-4a76-b4c0-b59b2656692d)

This is the information provided by the official website
![image](https://github.com/minio/minio/assets/86996849/779fb3de-2fa9-4130-9041-495272ba3df8)
Isn't it supposed to be 4 by default? Why is mine 1

**Comment by klauspost** [comment, marked as answer]
EC is global across the cluster.
